docs/user/00-50-limitations.md

```diff
@@ -48,11 +48,11 @@ The shortest build time (the limit) is approximately 15 seconds and requires no
 
 Running multiple Function build jobs at once (especially with no limits) may drain the cluster resources. To mitigate such risk, there is an additional limit of 5 simultaneous Function builds. If a sixth one is scheduled, it is built once there is a vacancy in the build queue.
 
-This limitation is configurable using [`containers.manager.envs.functionBuildMaxSimultaneousJobs`](../../../05-technical-reference/00-configuration-parameters/svls-01-serverless-chart.md#configurable-parameters).
+This limitation is configurable using [`containers.manager.envs.functionBuildMaxSimultaneousJobs`](/docs/user/05-10-chart.md#configurable-parameters).
```

Comment:
@kwiatekus Since we remove the chart doc, should I remove this sentence or only the link?

Reply:
Please remove the sentence. Once we have it configurable via Serverless CR we can add explenation how to configure in the new way